### PR TITLE
Bug 1639196 - add user_activity script

### PR DESCRIPTION
**Pull request by akkomar**
This adds a job for exporting user_activity aggregates to S3.

### CODE REVIEW

File: scripts/public_data_report_user_activity.sql [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        days_since_seen = 0
+    GROUP BY
+        client_id,
+        country,
+        week_start
+    HAVING
+        avg_hours_usage_daily_per_user < 24 --remove outliers
+),
+daily_usage AS (
+    SELECT
```
Comment (fbertsch) [inline scripts/public_data_report_user_activity.sql:83]: Looks like the above approach could simplify all of these.

File: scripts/public_data_report_user_activity.sql [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    WHERE
+        is_last_day_of_week
+        AND days_since_seen<7
+),
+latest_releases AS (
+    SELECT
+        MAX(SPLIT(build.target.version, '.')[OFFSET(0)]) AS latest_major_version,
+        DATE(build.build.date) AS day
+    FROM
+        `moz-fx-data-shared-prod.telemetry.buildhub2`
```
Comment (fbertsch) [inline scripts/public_data_report_user_activity.sql:184]: Awesome! What a good use of buildhub data.

File: scripts/public_data_report_user_activity.sql [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+    mau.mau,
+    daily_usage.avg_hours_usage_daily,
+    intensity.intensity,
+    new_profile_rate.new_profile_rate,
+    latest_version_ratio.latest_version_ratio
+FROM
+    mau
+JOIN
+    daily_usage
+ON
```
Comment (fbertsch) [inline scripts/public_data_report_user_activity.sql:248]: Nit: couldn't all of these use `USING(week_start, country_name)`?

File: scripts/backfill_public_data_report_user_activity.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -o xtrace
+
+# This script is used to bootstrap aggregate user_activity table (weekly query is maintaned in bigquery-etl repository)
```
Comment (fbertsch) [inline scripts/backfill_public_data_report_user_activity.sh:4]: Can you link to that here?

File: scripts/public_data_report_user_activity.sql [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+        country,
+        subsession_hours_sum,
+        days_seen_bits,
+        days_created_profile_bits,
+        client_id,
+        app_version
+    FROM
+        `moz-fx-data-shared-prod.telemetry.clients_last_seen`
+    WHERE
+        submission_date >= '2018-12-31'
```
[the remark this answers is not in the document]
Comment (akkomar) [reply scripts/public_data_report_user_activity.sql:16]: No, this query is used just for populating this table from scratch.

File: scripts/public_data_report_user_activity.sql [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+                 'France',
+                 'Germany',
+                 'India',
+                 'Indonesia',
+                 'Italy',
+                 'Poland',
+                 'Russia',
+                 'United States')
+),
+mau AS (
```
[the remark this answers is not in the document]
Comment (akkomar) [reply scripts/public_data_report_user_activity.sql:36]: Very nice, thanks!

**Pull request merged**